ParcelDistributionCenter/ParcelDistributionCenter.Model/Models/Courier.cs

```diff
@@ -3,8 +3,8 @@
 namespace ParcelDistributionCenter.Model.Models
 {
     public class Courier
-
-    {
+    { 
+        [JsonConstructor]
```

Comment:
Po co się to daje z ciekawości?

Reply:
Jak jest kilka konstruktorów, to funkcja tworząca obiekty z JSONa musi wiedzieć z którego konstruktora skorzystać.